### ISSUE TITLE
access token is invalid

### ISSUE THREAD

**Issue by cm0n** [austinlyons/dcjs-leaflet-untappd#3]
Hello,

 many thanks for this brilliant illustration and explanation.
Only in the example page http://austinlyons.github.io/dcjs-leaflet-untappd/ the access token is not more valid.

best regards

**Comment by austinlyons** [comment]
Hey there cm0n, yes you'll need to plug in your own Mapbox token, mine used up all of it's free views.

**Comment by cm0n** [comment]
Yes, I know I need own token, I only wanted to tell the token on the demo page is no longer valid. At first glance it looks how it is from the side does not work, many leave the site then probably again because it's not just partially.

For me it was so until I checked later something exactly as I could tell no more the token is valid.

You can close this issue if you mean that the demo page  need no valid token.

Best regards
cm0n

**Comment by austinlyons** [comment]
@cm0n I'll leave the issue here so others can see that they'll need to plug in their own token and then the map will work for them. I don't want anyone to spend too much time trying to figure out why that feature seems broken. Thanks for bringing it up.

**Comment by braaannigan** [comment]
You can get the map to work by changing the L.tileLayer block in index.html to use openstreetmap:
  L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
      // id: '<your id here>',
      // accessToken: '<your token here>',
      maxZoom: 16
    }).addTo(map);
